### DISCUSSION TITLE
is this package work in react native app

### DISCUSSION THREAD

**Discussion by nurfync** [roerohan/react-vnc#32] [Q&A] [answered]
Hi i am trying to control rasberry pi 3 from my react native app and i need a package for that.

**Comment by roerohan** [comment, marked as answer]
@nurfync does it throw an error in react native?
Can you describe what's not working?

**Reply by nurfync** [reply]
![Ekran Resmi 2022-04-06 13 38 38](https://user-images.githubusercontent.com/83401951/161957055-f8653a7f-4875-41e8-87d7-19e45c5c1bb7.png)

**Reply by roerohan** [reply]
There seems to be an issue in the React version. Maybe try downgrading the react version to React 17?

**Reply by roerohan** [reply]
Hey, just ran the library with `React 18` and it seems to be working fine! The issue is somewhere In your dependencies, where the `nvcdisplay` package depends on `react@17.0.2` whereas you're using `react@18.0.0`.

Maybe try removing your `node_modules` and `package-lock.json` file and re-running `npm install`?

As you can see here, it works perfectly with `react@18.x.x`
![image](https://user-images.githubusercontent.com/42958812/181907891-6df6ad36-8c01-48d8-9aef-fee9f1cfba62.png)

**Reply by nurfync** [reply]
Thank you for your answer. I will try it when i'm working on this project again.
